src/AutoRest.CSharp/LowLevel/Output/OperationMethodChainBuilder.cs

```diff
@@ -94,12 +102,41 @@ public LowLevelClientMethod BuildOperationMethodChain()
             return new LowLevelClientMethod(protocolMethodSignature, convenienceMethod, _restClientMethod, requestBodyType, responseBodyType, diagnostic, _protocolMethodPaging, Operation.LongRunning, _conditionHeaderFlag);
         }
 
-        private bool ShouldConvenienceMethodGenerated(ReturnTypeChain returnTypeChain)
+        private bool ShouldGenerateConvenienceMethod()
```

Comment:
Could we add some comments to explain the conditions we are using here?

Reply:
It is self-explaint.